### ISSUE TITLE
elementary method functions for multiple imputation with mice

### ISSUE THREAD

**Issue by avehtari** [stan-dev/rstanarm#164]
#### Summary:
This is an idea for a "new developer friendly" project.
mice (Multivariate Imputation by Chained Equations) package
https://cran.r-https://cran.r-project.org/package=mice
allows user defined elementary imputation methods. It would be quite easy to make functions which would allow use of rstanarm, too.

#### Description:
Implement Stan versions of elementary imputation methods
- [ ] mice.impute.stan.norm
- [ ] mice.impute.stan.logreg
- [ ] mice.impute.stan.polr
- [ ] mice.impute.stan.2l.norm
- [ ] mice.impute.stan.2lonly.norm
based on the corresponding existing functions (without .stan in the name). The existing norm function uses a fixed conjugate prior and analytic posterior, logreg uses glm.fit, polr uses a function from MASS package, and 2l functions use Gibbs sampling. rstanarm would provide more flexibility on priors and better inference.

**Comment by bgoodri** [comment]
Yeah, there was a postdoc at Columbia who was doing these things for the
mice and mi packages. One difficulty is asking the users for priors when
each variable with missingness has a turn at being the "outcome". Another
is the MCMC is a bit slow for this, so they were doing optimization, which
is not quite as reliable.

**Comment by avehtari** [comment]
I know people who used Stan for mice style imputation using their own code for chaining. They didn't care it was slow, they wanted the best possible inference. I know there can be also problems of checking all diagnostics for each model fit etc.
